bundle/README.md

```diff
@@ -152,3 +152,111 @@ To configure `bookinfo` with a gateway using `Gateway API`:
     $ echo "http://${GATEWAY_URL}/productpage"
     ```
 
+## Istio Addons Integrations
+
+Istio can be integrated with other software to provide additional functionality (More information can be found in: https://istio.io/latest/docs/ops/integrations/). The Sail Operator provides support for the following integrations:
```

Comment:
As we don't really provide "support" for any of this... we probably shouldn't say "provides suport", but rather... "The following addons are for demonstration or development purposes only and should not be used in production environments."